x-pack/plugins/apm/public/components/routing/mobile_service_detail/index.tsx

```diff
@@ -30,7 +30,7 @@ export function page({
   searchBarOptions,
 }: {
   title: string;
-  tab: React.ComponentProps<typeof MobileServiceTemplate>['selectedTab'];
+  tab: React.ComponentProps<typeof MobileServiceTemplate>['selectedTabKey'];
```

Comment:
Should this also be renamed then?

```suggestion
  tabKey: React.ComponentProps<typeof MobileServiceTemplate>['selectedTabKey'];
```

Reply:
https://github.com/elastic/kibana/pull/152440/commits/81057011250905361e78f72df993b359079068b7